### PR TITLE
HTCONDOR-2228 use ad-attr for all non-unique classad attributes

### PR DESCRIPTION
**Pull request by GregThain**
<Insert PR description here, and leave checklist below for code review.>

# HTCondor Pull Request Checklist for internal reviewers

- [x] Verify that (GitHub thinks) the merge is clean. If it isn't, and you're confident you can resolve the conflicts, do so. Otherwise, send it back to the original developer.
- [x] Verify that the related Jira ticket exists and has a target version number and that it is correct.
- [x] Verify that the Jira ticket is in review status and is assigned to the reviewer.
- [x] Verify that the Jira ticket (HTCONDOR-xxx) is mentioned at the beginning of the title. Edit it, if not
- [x] Verify that the branch destination of the PR matches the target version of the ticket
- [x] Check for correctness of change
- [ ] Check for regression test(s) of new features and bugfixes (if the feature doesn't require root)
- [x] Check for documentation, if needed  (documentation [build logs](https://readthedocs.org/projects/htcondor/builds/))
- [ ] Check for version history, if needed
- [ ] Check BaTLab dashboard for successful build (https://batlab.chtc.wisc.edu/results/workspace.php) and test for either the PR or a workspace build by the developer that has the Jira ticket as a comment.
- [x] Check that each commit message references the Jira ticket (HTCONDOR-xxx)

## After the above
- Hit the merge button if the pull request is approved and it is not a security patch (security changes require 2 additional reviews)
- If the pull request is approved, take the ticket out of review state
- Assign JIRA Ticket back to the developer

### CODE REVIEW

File: docs/admin-manual/ap-policy-configuration.rst [hunk 1]
```diff
@@ -114,7 +114,7 @@ terminates further evaluation of other submit requirements, and is the
 only requirement reported. Each submit requirement is evaluated in the
 context of the *condor_schedd* ClassAd, which is the ``MY.`` name space
 and the job ClassAd, which is the ``TARGET.`` name space. Note that
-``JobUniverse`` and ``RequestMemory`` are both job ClassAd attributes.
+``JobUniverse`` and :ad-attr:`RequestMemory` are both job ClassAd attributes.
```
Comment (ColeBollig) [inline docs/admin-manual/ap-policy-configuration.rst:117]:
```suggestion
:ad-attr:`JobUniverse` and :ad-attr:`RequestMemory` are both job ClassAd attributes.
```

File: docs/admin-manual/cm-configuration.rst [hunk 2]
```diff
@@ -244,27 +244,27 @@ entries` section for definitions of these configuration variables.
     accounting group, with a defined group quota, then this attribute is
     the integer number of slots currently utilized by the group.
 
-:index:`RemoteGroup<single: RemoteGroup; ClassAd attribute, ephemeral>`\ ``RemoteGroup``
+:index:`RemoteGroup<single: RemoteGroup; ClassAd attribute, ephemeral>`\ :ad-attr:`RemoteGroup`
     The accounting group name of the owner of the currently running job.
 
 :index:`RemoteGroupQuota<single: RemoteGroupQuota; ClassAd attribute, ephemeral>`\ ``RemoteGroupQuota``
```
Comment (ColeBollig) [inline docs/admin-manual/cm-configuration.rst:250]:
```suggestion
:index:`RemoteGroupQuota<single: RemoteGroupQuota; ClassAd attribute, ephemeral>`\ :ad-attr:`RemoteGroupQuota`
```

File: docs/admin-manual/ap-policy-configuration.rst (hunk 1 above)
Comment (GregThain) [reply docs/admin-manual/ap-policy-configuration.rst:117]: Note that `JobUniverse` is not :classad-attribute-defined: in classad-attributes/job-classad-attributes.rst.

File: docs/admin-manual/ep-policy-configuration.rst [hunk 3]
```diff
@@ -958,8 +958,8 @@ The most important expression to the *condor_startd* is the
 :macro:`START` expression. This expression describes the
 conditions that must be met for a machine or slot to run a job. This
 expression can reference attributes in the machine's ClassAd (such as
-``KeyboardIdle`` and ``LoadAvg``) and attributes in a job ClassAd (such
-as ``Owner``, ``Imagesize``, and ``Cmd``, the name of the executable the
+:ad-attr:`KeyboardIdle` and :ad-attr:`LoadAvg`) and attributes in a job ClassAd (such
+as :ad-attr:`Owner`, ``Imagesize``, and :ad-attr:`Cmd`, the name of the executable the
```
Comment (ColeBollig) [inline docs/admin-manual/ep-policy-configuration.rst:962]:
```suggestion
as :ad-attr:`Owner`, :ad-attr:`ImageSize`, and :ad-attr:`Cmd`, the name of the executable the
```

File: docs/admin-manual/ep-policy-configuration.rst [hunk 4]
```diff
@@ -983,7 +983,7 @@ a match, or against whatever request ClassAd currently has claimed the
 machine. However, by locally evaluating an expression, the machine only
 evaluates the expression against its own ClassAd. If an expression
 cannot be locally evaluated (because it references other expressions
-that are only found in a request ClassAd, such as ``Owner`` or
+that are only found in a request ClassAd, such as :ad-attr:`Owner` or
 ``Imagesize``), the expression is (usually) undefined. See
```
Comment (ColeBollig) [inline docs/admin-manual/ep-policy-configuration.rst:987]:
```suggestion
:ad-attr:`ImageSize`), the expression is (usually) undefined. See
```

File: docs/admin-manual/ap-policy-configuration.rst (hunk 1 above)
Comment (GregThain) [reply docs/admin-manual/ap-policy-configuration.rst]: And fixed...

File: docs/classads/print-formats.rst [hunk 5]
```diff
@@ -222,87 +222,87 @@ value of certain attributes, often multiple attributes. The list for :tool:`cond
...
-   ``JobUniverse``, ``ClusterId``, ``DAGManJobId``, and ``DAGNodeName``.
+   :ad-attr:`JobUniverse`, :ad-attr:`ClusterId`, :ad-attr:`DAGManJobId`, and ``DAGNodeName``.
 
 ``BUFFER_IO_MISC``
    Used for the ``MISC`` column of the ``-io`` output of :tool:`condor_q`.
-   This function constructs an IO string that varies by ``JobUniverse``.
+   This function constructs an IO string that varies by :ad-attr:`JobUniverse`.
    The output for Standard universe jobs refers to ``FileSeekCount``, ``BufferSize`` and ``BufferBlockSize``.
-   For all other jobs it refers to ``TransferrringInput``, ``TransferringOutput`` and ``TransferQueued``.
+   For all other jobs it refers to ``TransferrringInput``, :ad-attr:`TransferringOutput` and :ad-attr:`TransferQueued`.
```
Comment (ColeBollig) [inline docs/classads/print-formats.rst:233]:
```suggestion
   For all other jobs it refers to :ad-attr:`TransferringInput`, :ad-attr:`TransferringOutput` and :ad-attr:`TransferQueued`.
```

File: docs/classads/print-formats.rst [hunk 6]
```diff
@@ -222,87 +222,87 @@ value of certain attributes, often multiple attributes. The list for :tool:`cond
...
+   This function constructs a batch name string using value of the :ad-attr:`JobBatchName`
    attribute if it exists, otherwise it constructs a batch name from
-   ``JobUniverse``, ``ClusterId``, ``DAGManJobId``, and ``DAGNodeName``.
+   :ad-attr:`JobUniverse`, :ad-attr:`ClusterId`, :ad-attr:`DAGManJobId`, and ``DAGNodeName``.
 
 ``BUFFER_IO_MISC``
    Used for the ``MISC`` column of the ``-io`` output of :tool:`condor_q`.
-   This function constructs an IO string that varies by ``JobUniverse``.
+   This function constructs an IO string that varies by :ad-attr:`JobUniverse`.
    The output for Standard universe jobs refers to ``FileSeekCount``, ``BufferSize`` and ``BufferBlockSize``.
```
Comment (ColeBollig) [inline docs/classads/print-formats.rst:232]: Is standard universe even still a thing?

File: docs/classads/print-formats.rst [hunk 7]
```diff
@@ -314,23 +314,23 @@ value of certain attributes, often multiple attributes. The list for :tool:`cond
...
-   Use this for Job attributes that are valued in Mb, such as ``MemoryUsage``.
+   Use this for Job attributes that are valued in Mb, such as :ad-attr:`MemoryUsage`.
 
 ``REMOTE_HOST``
    Used for the ``HOST(S)`` column of the ``-run`` output of :tool:`condor_q`.
-   This function extracts the host name from a job attribute appropriate to the ``JobUniverse`` value of the job.
+   This function extracts the host name from a job attribute appropriate to the :ad-attr:`JobUniverse` value of the job.
    For Local and Scheduler universe jobs, the Schedd that was queried is used using a variable internal to :tool:`condor_q`.
-   For grid uiniverse jobs, the ``EC2RemoteVirtualMachineName`` or ``GridResources`` attributes are used.
+   For grid universe jobs, the :ad-attr:`EC2RemoteVirtualMachineName` or ``GridResources`` attributes are used.
```
Comment (ColeBollig) [inline docs/classads/print-formats.rst:327]:
```suggestion
   For grid universe jobs, the :ad-attr:`EC2RemoteVirtualMachineName` or :ad-attr:`GridResource` attributes are used.
```

Comment (ColeBollig) [reply docs/classads/print-formats.rst:327]: Not sure if there is a GridResources (Plural)

File: docs/grid-computing/grid-universe.rst [hunk 8]
```diff
@@ -344,11 +344,11 @@ The following table summarizes how job ClassAd attributes will be translated int
...
 | Job ClassAd       | Slurm               |
 +===================+=====================+
-| ``RequestMemory`` | ``--mem``           |
+| RequestMemory`    | ``--mem``           |
 +-------------------+---------------------+
-| ``BatchRuntime``  | ``--time``          |
+| BatchRuntime`     | ``--time``          |
 +-------------------+---------------------+
-| ``BatchProject``  | ``--account``       |
+| BatchProject`     | ``--account``       |
```
Comment (ColeBollig) [inline docs/grid-computing/grid-universe.rst:351]: This Table is messed up

File: docs/admin-manual/cm-configuration.rst (hunk 2 above)
Comment (GregThain) [reply docs/admin-manual/cm-configuration.rst:250]: I want to think about where to put these.  `RemoteGroupQuota` only shows up in the negotiator's copy of the job ad, so I'm worried that defining in this job ad attribute section would give people the idea that it is available everywhere.

**Review by ColeBollig — APPROVED** [review]
LGTM

**Pull request merged**